telemetry/1.1/README.adoc

```diff
@@ -0,0 +1,49 @@
+---
+title: "MicroProfile Telemetry 1.1 (under development)"
+date: 2023-05-15
+summary: "Release for MicroProfile 6.1"
```

Comment:
We haven't committed to the next release being 6.1. I'd leave the summary blank, the same as Config 3.1.